packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.cnstyles.ts

```diff
@@ -10,7 +10,7 @@ import {
 import { IContextualMenuStyles, IMenuItemStyles } from './ContextualMenu.types';
 import { memoizeFunction } from '../../Utilities';
 
-const ContextualMenuItemHeight = '32px';
+export const ContextualMenuItemHeight = '32px';
```

Comment:
>ContextualMenuItemHeight [](start = 13, length = 24)

nit: as this is a symbolic const, it should use symbolic const notation (`CONTEXTUAL_MENU_ITEM_HEIGHT`)